cloud-info/Dockerfile

```diff
@@ -1,33 +1,65 @@
+FROM python:3 as build
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# hadolint ignore=DL3008
+RUN curl -s https://dist.eugridpma.info/distribution/igtf/current/GPG-KEY-EUGridPMA-RPM-3 \
+	| apt-key add - \
+    && echo "deb https://repository.egi.eu/sw/production/cas/1/current egi-igtf core" > /etc/apt/sources.list.d/igtf.list \
+    && apt-get update \
+    && apt-get install -y ca-policy-egi-core \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /cloud-info
+
+RUN python -m venv /cloud-info/venv
+ENV PATH="/cloud-info/venv/bin:$PATH"
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt \
+    && cat /etc/grid-security/certificates/*.pem >> "$(python -m requests.certs)"
+
+COPY . .
```

Comment:
Since the `build` container is only creating the `venv`, is this really required?

Later from the main image you only 
```
COPY --chown=python:python --from=build /cloud-info/venv ./venv
```